toxcore/tox.api.h

```diff
@@ -443,7 +455,9 @@ static class options {
    * @deprecated The memory layout of this struct (size, alignment, and field
    * order) is not part of the ABI. To remain compatible, prefer to use $new to
    * allocate the object and accessor functions to set the members. The struct
-   * will become opaque (i.e. the definition will become private) in v0.2.0.
+   * will become opaque (i.e. the definition will become private) in v0.2.x.
```

Comment:
API/ABI breaking change in `v0.2.x`? I thought that our versioning was that of Semantic Versioning but with the leading zero removed. Making such change in a `v0.2.x` violates that.